server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -15,16 +15,16 @@ roles = pbench-results
 
 [Indexing]
 index_prefix = container-pbench
-uri = http://localhost:9200
+uri = https://localhost:9200
 
 [database]
-uri = postgresql://pbenchcontainer:pbench@localhost:5432/pbenchcontainer
+uri = postgresql://pbenchcontainer:pbench@localhost:5432/pbenchcontainer?sslmode=require&sslcert=/srv/pbench/postgres.pem
```

Comment:
Are you sure we want `sslmode=require` and not `sslmode= verify-full` or `sslmode= verify-ca`?

I'm OK with `require` for functional test and even for IntLab deployment, but, for general purposes, if we're going to go to the trouble of using SSL, it seems like we might as well make sure that the server we're connecting to is the one that we expect.